src/main/java/com/crowdin/client/ai/model/AiReportDownloadLinkResponseModel.java

```diff
@@ -0,0 +1,11 @@
+package com.crowdin.client.ai.model;
+
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+public class AiReportDownloadLinkResponseModel {
```

Comment:
Use the existing `java/com/crowdin/client/core/model/DownloadLink.java` class instead of creating a new one